plugin.js

```diff
@@ -74,7 +74,9 @@ Plugin.prototype.exec = function (server, cb) {
   }
 
   try {
-    this.server = this.parent.override(server, func, this.opts)
+    if (!this.isAfter || this.isAfter === 3) {
```

Comment:
I think we should just check if isAfter is true.

Reply:
The failing errors are the same